tests/cpp/aot/llvm/field_cgraph_test.py

```diff
@@ -0,0 +1,17 @@
+import argparse
+
+from utils import compile_field_aot
+
+import taichi as ti
+
+parser = argparse.ArgumentParser()
+parser.add_argument("--arch", type=str)
+args = parser.parse_args()
+
+if __name__ == "__main__":
+    if args.arch == "cpu":
+        compile_field_aot(arch=ti.cpu, compile_for_cgraph=True)
```

Comment:
aha I feel like this can be merged with `field_aot_test.cpp` with an additional command argument? ;)

Reply:
sounds good~